### PR TITLE
adding aioconsole

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timkpaine** [comment]
@vxgmichel would you like to be listed as a maintainer?

**Comment by vxgmichel** [comment]
@timkpaine 
> would you like to be listed as a maintainer?

Sure thanks!

File: recipes/aioconsole/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "aioconsole" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7c038bb40b7690bf5be6b17154830b7bff25e7be1c02d8420a346c3efbd5d8e5
```
Comment (vxgmichel) [inline recipes/aioconsole/meta.yaml:10]: Shouldn't the hash be parametrize as well?
```
{% set sha256 = "7c038bb40b7690bf5be6b17154830b7bff25e7be1c02d8420a346c3efbd5d8e5" %}
```

**Comment by timkpaine** [comment]
@conda-forge-admin, please rerender

**Comment by timkpaine** [comment]
@conda-forge/help-python all set

**Review by dopplershift — APPROVED** [review]

**Comment by timkpaine** [comment]
thanks @dopplershift!

**Pull request merged**